shared/presentation/src/commonMain/kotlin/network/bisq/mobile/presentation/BasePresenter.kt

```diff
@@ -225,4 +226,13 @@ abstract class BasePresenter(private val rootPresenter: MainPresenter?): ViewPre
     }
 
     private fun isRoot() = rootPresenter == null
+
+    companion object {
+        lateinit var strings: AppStrings
+    }
+
+    fun setStrings(localStrings: AppStrings) {
```

Comment:
this works because kotlin automatic generation of get/set based on public variables is applied to the componion object ( BasePresenter:: ) instead of the object instance. But generally using get/set methods in Kotlin is dicouraged in favour of public val/var s.

Reply:
Got it. I will remove the `setStrings()` and try to assign value to `strings` directly in next PR